ethereum-consensus/src/deneb/polynomial_commitments.rs

```diff
@@ -0,0 +1,147 @@
+use crate::{deneb::blob_sidecar::Blob, primitives::Bytes32, ssz::prelude::*};
+pub use c_kzg::KzgSettings;
+use std::fmt;
+use thiserror::Error;
+
+pub const BYTES_PER_FIELD_ELEMENT: usize = 32;
+pub const BYTES_PER_COMMITMENT: usize = 48;
+pub const BYTES_PER_PROOF: usize = 48;
+
+pub type VersionedHash = Bytes32;
+pub type FieldElement = Bytes32;
+pub type KzgCommitment = ByteVector<BYTES_PER_COMMITMENT>;
+pub type KzgProof = ByteVector<BYTES_PER_PROOF>;
+
+#[derive(Debug, Error)]
+pub struct Error(c_kzg::Error);
+
+impl fmt::Display for Error {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        // Implement custom formatting logic here
+        write!(f, "Custom error: {:?}", self.0)
+    }
+}
```

Comment:
I think we can just drop this?

Reply:
`error[E0277]: `polynomial_commitments::Error` doesn't implement `std::fmt::Display``